### ISSUE TITLE
avatar and name missing in group chat 

### ISSUE THREAD

**Issue by Nokius** [QtGram/SailorGram-BETA#14]
in the new Beta client, in one chatroom one person has no avatar and no name 

Yes the person has a avatar (if the person has no avatar you see normally the empty circle] 
Please take a look on the attached screenshots 

[avatar_issue_oldstable_clinet](https://cloud.githubusercontent.com/assets/879178/20439407/9eb2102a-adbb-11e6-86ff-0a2c43f7addb.png)
[avatar_issue_beta_clinet](https://cloud.githubusercontent.com/assets/879178/20439381/7f4f39c4-adbb-11e6-9927-8cfc67572ce9.png)

**Comment by Dax89** [comment]
Yes it's a known issue, the library still have some cache issues :)

**Comment by ferlanero** [comment]
Avatar issues still on beta 4
![captura_20161128_005](https://cloud.githubusercontent.com/assets/1192395/20675344/3e635426-b58c-11e6-8f34-e5b4234ab07d.png)

**Comment by Tichy** [comment]
How about removing also the nickname? To save more space.

**Comment by Dax89** [comment]
@Tichy: Yes, I've missed to include that in beta 4 because it was asked few weeks ago :)

**Comment by Nokius** [comment]
@Dax89 what about having an option so the user can decide to see the username in group chats.

**Comment by Dax89** [comment]
@Nokius: Yes, that's another boolean :P

**Comment by TomiOhl** [comment]
Avatars are supposed to appear only in normal groups right? Cause they're missing in supergroups for me. 
BTW I've installed Sailfish 2.0.5.6 on my main phone Xiaomi Mi3, I'm planning to use it at least for a week. I hope I won't miss messages before that other issue (I'll wait for sync end, this should be the workaround afaik).
